package.json

```diff
@@ -55,8 +59,13 @@
     "yamljs": "^0.3.0"
   },
   "devDependencies": {
+    "@types/jest": "^25.2.1",
+    "fsevents": "^1.2.9",
```

Comment:
I got an error related to fsevents `Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})`

Reply:
yeah fsevent is only for mac... this was to lock down a version to support local dev on mac, it should run without it on linux though eh?

Reply:
yea it does run, was just pointing it out cause I have seen `npm i` complain about this before